src/bower/PackageManager.js

```diff
@@ -428,14 +451,22 @@ define(function (require, exports) {
         return Bower.listCache(ConfigurationManager.getConfiguration());
     }
 
-    exports.install                 = install;
-    exports.uninstall               = uninstall;
-    exports.installFromBowerJson    = installFromBowerJson;
-    exports.prune                   = prune;
-    exports.update                  = update;
-    exports.info                    = info;
-    exports.search                  = search;
-    exports.listCache               = listCache;
-    exports.list                    = list;
-    exports.packagesWithVersions    = packagesWithVersions;
+    /**
+     * @return {boolean}
+     */
+    function isModificationInProgress() {
+        return _isModificationInProgress;
+    }
+
+    exports.install                  = _wrapModification(install);
+    exports.uninstall                = _wrapModification(uninstall);
+    exports.installFromBowerJson     = installFromBowerJson;
```

Comment:
Perhaps this should be called `install` and `install` should be called `installByName`. Then "peer" functions (such as `uninstall` should have "ByName" suffix).